src/components/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./Box";
+export * from "./icons";
```

Comment:
**suggestion**: to be consistent in naming maybe it should be capitalized (`./Icons`)?

Reply:
Do we want ot use capitilize icons in import like this 
`@saleor/macaw-ui/components/icons/ArrowUpIcon` ?